tests/solvers/linear/sinkhorn_lr_test.py

```diff
@@ -25,7 +25,8 @@
 class TestLRSinkhorn:
 
   @pytest.fixture(autouse=True)
-  def initialize(self, rng: jnp.ndarray):
+  def initialize(self):
+    rng = jax.random.PRNGKey(0)
```

Comment:
Please revert this, the `rng` fixture is defined in `tests/conftest.py` exactly like this.
Think we need to pass some `rng` when calling the solver in that flaky test.

Reply:
I am not sure why this would change anything, since, IIUC, we are passing `None` to `kwarg_inits`, resulting in a `PRNGKey(0)` passed on by default, and therefore no randomness, is there anything I am missing. 

Additionally, if I pass `kwarg_inits = {'key' : jax.random.PRNGKey(0)}` I do get an (obscure) error.

Reply:
You're right, it wouldn't make sense, also because it fails for `rank2` initializer, which doesn't make use of the random key (side note: the `key` needs to be passedn when calling the solver).
Locally on my Mac, this test seems to be passing, looks platform specific... I propose to change the test to:
```python
if solved.converged:
  assert criterions[-1] < threshold
```